flink-docs/src/main/java/org/apache/flink/docs/rest/RestAPIDocGenerator.java

```diff
@@ -258,6 +265,37 @@ private static String createMessageHtmlEntry(Class<?> messageClass, Class<?> emp
 		return json;
 	}
 
+	/**
+	 * Create character escapes for HTML when generating JSON request/response string.
+	 */
+	private static class HTMLCharacterEscapes extends CharacterEscapes {
```

Comment:
It would be good to document that this is necessary because the `id` field of the schema may contain generic types, like `SerializedValue<Object>`.

Reply:
good point. added in comments to illustrate the necessity.